client/src/components/shared/ViewTitle/ViewTitle.module.scss

```diff
@@ -4,12 +4,13 @@
   width: 100%;
   height: 8.8rem;
   color: $color-octant-dark;
-  font-size: $font-size-24;
+  @include fontBig();
```

Comment:
`fontBig`, but font-size-24? I would introduce variants to `fontBig` mixin and use as few overrides as possible, it can get out of hand pretty easily. WDYT?

Reply:
```
@mixin fontBig($font-size: $font-size-32) {
  font-family: 'Inter Tight', sans-serif;
  font-size: $font-size;
}
```
Why can't we just pass `font-size` here as parameter? It's already defined, right?

Reply:
This is what i wrote, remove default param `@mixin fontBig($font-size: $font-size-32)` -> `@mixin fontBig($font-size)` and pass fontSize every time :)

Reply:
OK.

Reply:
@aziolek done